docs/BuildFromSource.md

```diff
@@ -1,151 +1,185 @@
 
 ```ps1
 git clone --recursive https://github.com/dotnet/aspnetcore
 ```
 
-To update an existing copy, run:
+If you've already cloned the aspnetcore repo without fetching subdmoule sources, you can fetch them after cloning by running the following command.
 
 ```ps1
 git submodule update --init --recursive
 ```
 
-## Install pre-requisites
+> :bulb: Some ISPs have been know to use web filtering software that has caused issues with git repository cloning, if you experience issues cloning this repo please review <https://help.github.com/en/github/authenticating-to-github/using-ssh-over-the-https-port>.
 
-### Windows
+## Step 2: Install pre-requisites
 
-Building ASP.NET Core on Windows requires:
+Developing the aspnetcore repo requires some additional tools to build the source code and run integration tests.
 
-* Windows 10, version 1803 or newer
-* At least 10 GB of disk space and a good internet connection (our build scripts download a lot of tools and dependencies)
-* Visual Studio 2019. <https://visualstudio.com>
-  * To install the exact required components, run [eng/scripts/InstallVisualStudio.ps1](/eng/scripts/InstallVisualStudio.ps1).
+### On Windows
 
-    ```ps1
-    PS> ./eng/scripts/InstallVisualStudio.ps1  [-Edition {Enterprise|Community|Professional}]
-    ```
+Building ASP.NET Core on Windows (10, version 1803 or newer) requires that you have the following tooling installed.
 
-    However, any Visual Studio 2019 instance that meets the requirements should be fine. See [global.json](/global.json)
-    and [eng/scripts/vs.json](/eng/scripts/vs.json) for those requirements. By default, the script will install Visual Studio Enterprise Edition, however you can use a different edition by passing the `-Edition` flag.
-    Even if you have installed Visual Studio, still recommend you should use this script to install again to avoid errors due to missing components just in case.
-* Git. <https://git-scm.org>
-* NodeJS. LTS version of 10.14.2 or newer <https://nodejs.org>.
-* Install yarn globally (`npm install -g yarn`)
-* Java Development Kit 11 or newer. Either:
-  * OpenJDK <https://jdk.java.net/>
-  * Oracle's JDK <https://www.oracle.com/technetwork/java/javase/downloads/index.html>
-  * To install a version of the JDK that will only be used by this repo, run [eng/scripts/InstallJdk.ps1](/eng/scripts/InstallJdk.ps1)
+> :bulb: Be sure you have least 10 GB of disk space and a good Internet connection. The build scripts will download several tools and dependencies onto your machine.
+
+**[Visual Studio 2019](https://visualstudio.com)**
+
+Visual Studio 2019 (16.8) is required to build the repo locally. If you don't have visual studio installed you can run [eng/scripts/InstallVisualStudio.ps1](/eng/scripts/InstallVisualStudio.ps1) to install the exact required dependencies.
+
+> :bulb: By default, the script will install Visual Studio Enterprise Edition, however you can use a different edition by passing the `-Edition` flag.
```

Comment:
Suggest adding another `:bulb:` that mentions `-Channel Preview` to install the very latest. This may be necessary at the moment because 16.8 remains in preview.